Drivers/transceiverConfig/TransceiverConfig.py

```diff
@@ -24,36 +24,6 @@ def writeData(self, input):
     #response = ser.read(128)
     ser.close()
     #return response
-  
-  def setBeaconOn(self):
```

Comment:
I went back and modified this in a later commit. We can leave in setBeaconOn and setBeaconOff so that the test for the transceiver will still work.